magic-link/magic-links-default-filters.php

```diff
@@ -217,3 +217,21 @@ function dt_magic_link_continue( bool $response, array $args ){
 /**
  * LINK EXPIRY CHECKER
  */
+
+
+/**
+ * GLOBAL NAME
+ */
+
+add_filter( 'dt_magic_link_global_name', 'dt_magic_link_global_name', 10, 1 );
+function dt_magic_link_global_name( $global_name ) {
+    if ( Disciple_Tools_Bulk_Magic_Link_Sender_API::option_exists( Disciple_Tools_Bulk_Magic_Link_Sender_API::$option_dt_magic_links_global_name_enabled ) && boolval( Disciple_Tools_Bulk_Magic_Link_Sender_API::fetch_option( Disciple_Tools_Bulk_Magic_Link_Sender_API::$option_dt_magic_links_global_name_enabled ) ) === true ) {
```

Comment:
@kodinkat this section will require 4 database calls. 2 is enough.
just use `boolval` and  `get_option( Disciple_Tools_Bulk_Magic_Link_Sender_API::$option_dt_magic_links_global_name_enabled, $default )`
and then get_option again with a default set to ''

Reply:
replace `Disciple_Tools_Bulk_Magic_Link_Sender_API::fetch_option()` with the wp function. the wrapper seems unnecessary